src/modules/quizzes.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable object-curly-newline, no-underscore-dangle */
+const qs = require('qs');
+const fetchPonyfill = require('fetch-ponyfill');
+const Promise = require('es6-promise');
+const EventDispatcher = require('../utils/event-dispatcher');
+const helpers = require('../utils/helpers');
+
+// Create URL from supplied quizId and parameters
+function createQuizUrl(quizId, parameters, options, path) {
+  const {
+    apiKey,
+    clientId,
+    sessionId,
+    segments,
+    userId,
+    version,
+  } = options;
+  const serviceUrl = 'https://quizzes.cnstrc.com';
+  let queryParams = { c: version };
+  let answersParamString = '';
+
+  queryParams.key = apiKey;
+  queryParams.i = clientId;
+  queryParams.s = sessionId;
+
+  // Pull user segments from options
+  if (segments && segments.length) {
+    queryParams.us = segments;
+  }
+
+  // Pull user id from options
+  if (userId) {
+    queryParams.ui = userId;
+  }
+
+  // Validate quiz id is provided
+  if (!quizId || typeof quizId !== 'string') {
+    throw new Error('quizId is a required parameter of type string');
+  }
+
+  if (path === 'finalize' && (typeof parameters.a !== 'object' || !Array.isArray(parameters.a) || parameters.a.length === 0)) {
+    throw new Error('a is a required parameter of type array');
+  }
+
+  if (parameters) {
+    const { section, a, versionId } = parameters;
+
+    // Pull section from parameters
+    if (section) {
+      queryParams.section = section;
+    }
+
+    // Pull version_id from parameters
+    if (versionId) {
+      queryParams.version_id = versionId;
+    }
+
+    // Pull a from parameters and transform
+    if (a) {
+      a.forEach((ans) => {
+        answersParamString += `&${qs.stringify({ a: ans }, { arrayFormat: 'comma' })}`;
+      });
+    }
+  }
+
+  queryParams._dt = Date.now();
+  queryParams = helpers.cleanParams(queryParams);
+
+  const queryString = qs.stringify(queryParams, { indices: false });
+
+  return `${serviceUrl}/v1/quizzes/${encodeURIComponent(quizId)}/${encodeURIComponent(path)}/?${queryString}${answersParamString}`;
+}
+
+/**
+ * Interface to quiz related API calls
+ *
+ * @module quizzes
+ * @inner
+ * @returns {object}
+ */
+class Quizzes {
+  constructor(options) {
+    this.options = options || {};
+    this.eventDispatcher = new EventDispatcher(options.eventDispatcher);
+  }
+
+  /**
+   * Retrieve next quiz from api
+   *
+   * @function getNextQuiz
```

Comment:
Have you considered naming this `getNextQuestion` to make it consistent with the API?